src/features/common/Layout/UserLayout/UserLayout.tsx

```diff
@@ -299,7 +299,7 @@ const UserLayout = ({ children }: { children: ReactNode }) => {
         router.push(localizedPath('/login'));
       }
     }
-  }, [contextLoaded, user, router]);
+  }, [contextLoaded, user]);
```

Comment:
I think `router.asPath`  needs to be added as a dependency here. Is there any specific reason you haven’t added it?

Reply:
This is supposed to run just once, not every time the path changes.